pkg/tarmak/provider/amazon/tf_remote_state.go

```diff
@@ -385,3 +385,169 @@ func (a *Amazon) initRemoteStateKMS() error {
 
 	return nil
 }
+
+func (a *Amazon) deleteRemoteStateBucket() error {
+	svc, err := a.S3()
+	if err != nil {
+		return err
+	}
+
+	_, err = svc.DeleteBucket(&s3.DeleteBucketInput{
+		Bucket: aws.String(a.RemoteStateName()),
+	})
+
+	if err != nil {
+		ec2err, ok := err.(awserr.Error)
+		if ok && ec2err.Code() == "BucketNotEmpty" {
```

Comment:
if !ok || ec2err.Code() != "BucketNotEmpty" {
  return err
}

for indentation again